so_box_biogeo/diags/estNsq.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+from MITgcmutils import rdmds, densjmd95
+from hspython import loadgrid
```

Comment:
Would it be better to include the function `loadgrid` locally, rather than require users to go out and obtain the code for it separately?